Userland/Libraries/LibCore/File.h

```diff
@@ -105,6 +105,6 @@ class File final : public SeekableStream {
 
 AK_ENUM_BITWISE_OPERATORS(File::OpenMode)
 
-using BufferedFile = BufferedSeekable<File>;
+using BufferedInputFile = BufferedInputSeekable<File>;
```

Comment:
:hyperbikeshed:, but isn't this more of a `InputBufferedFile`? I'm not sure whether a potential reader would relate the `Input` to the "File" instead of the "Buffer", and whether they would assume that an "input file" means that it can't be used for output.

Same probably applies to `BufferedInputSeekable` (although the confusion is less prevalent there) and the later `Output` variants.

Reply:
Half :hyperbikeshed: but also half English rules I guess :sweat_smile:
I wasn't sure about it, so let's go with that if you prefer it